modules/library/src/main/scala/zio/elasticsearch/ElasticHighlight.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022 LambdaWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zio.elasticsearch
+
+import zio.Chunk
+import zio.elasticsearch.highlighting.{HighlightField, Highlights}
+import zio.json.ast.Json
+
+object ElasticHighlight {
+
+  def highlightSafe[S](field: Field[S, _], config: Map[String, Json] = Map.empty): Highlights =
+    Highlights(Chunk(HighlightField(field.toString, config)))
+
+  def highlight(field: String, config: Map[String, Json] = Map.empty): Highlights =
```

Comment:
Perhaps we could move these methods to `SearchRequest`, so that one could use `.highlight("name", ...)` instead of `.highlights(highlight("name",...))`

Reply:
Agreed not to do this, because configs need to be applicable globally to all highlights.